src/components/index/sections/TeamSection.tsx

```diff
@@ -27,14 +32,17 @@ export default function TeamSection() {
       justify="center"
       component="section"
       className={classes.container}>
-      <Typography variant="h5" component="h2" className={classes.heading}>
```

Comment:
Please undo this change as it was requested by the SEO team. We should only have H1 and H2 on the homepage. If we don't specify it on the `<Typography variant="h5">` we'll get `<h5>` instead.

We can also adjust that globally, but I'm still not convinced we should 
https://material-ui.com/components/typography/#changing-the-semantic-element